common/src/main/java/org/dash/wallet/common/transactions/filters/CoinsReceivedTxFilter.kt

```diff
@@ -28,6 +30,11 @@ open class CoinsReceivedTxFilter(
         private set
 
     override fun matches(tx: Transaction): Boolean {
+        if (tx.isEntirelySelf(bag) || tx.getValue(bag).signum() < 0) {
+            // Not an incoming transaction
+            return false
+        }
+
```

Comment:
I wonder how this got in here.

Reply:
During CoinJoin testing, we had an issue where one of the CJ transactions was detected as a CrowdNode transaction, so a fix was made.